pkgs/tools/misc/manga-cli/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenvNoCC
+, lib
+, fetchFromGitHub
+, makeWrapper
+, bash
+, img2pdf
+, zathura
+}:
+stdenvNoCC.mkDerivation {
+  pname = "manga-cli";
+
+  version = "1.0.5";
+
+  src = fetchFromGitHub {
+    owner = "7USTIN";
+    repo = "manga-cli";
+    rev = "a69fe935341eaf96618a6b2064d4dcb36c8690b5";
+    sha256 = "sha256-AnpOEgOBt2a9jtPNvfBnETGtc5Q1WBmSRFDvQB7uBE4=";
+  };
+
+  buildInputs = [ bash ];
+
```

Comment:
We only install a script, so there are no `buildInputs` or a `buildPhase` for that matter.
```suggestion
```